### PR TITLE
Added option to nexusIO to not interleave the file.

### PR DESCRIPTION
**Pull request by theboocock**
This was problematic when using seqmagick to perform some conversions, to phylogenetic
software that did not support interleaving mode, specifically popart and network.

This change will not alter current usage of the write alignment function, but it does
enable the option to have interleaved nexus files output.

### CODE REVIEW

File: Bio/AlignIO/NexusIO.py [hunk 1]
```diff
@@ -114,7 +114,10 @@ def write_alignment(self, alignment):
         #For larger alginments, interleave to avoid very long lines
         #in the output - something MrBayes can't handle.
         #TODO - Default to always interleaving?
-        n.write_nexus_data(self.handle, interleave=(columns > 1000))
+        if(interleave):
```
Comment (peterjc) [inline Bio/AlignIO/NexusIO.py]: @PamelaM's suggestion made sense to me too. It also solve the style issue with C like ``if(interleave):`` which should have been just ``if interleave:``

File: Bio/AlignIO/NexusIO.py [hunk 2]
```diff
@@ -114,7 +114,10 @@ def write_alignment(self, alignment):
         #For larger alginments, interleave to avoid very long lines
         #in the output - something MrBayes can't handle.
         #TODO - Default to always interleaving?
```
Comment (peterjc) [inline Bio/AlignIO/NexusIO.py]: Yes, the comment would need updating and the TODO removed.

**Comment by peterjc** [comment]
Apologies that I had forgotten about this for so long.

The change looks relatively easy to rebase / merge by hand, including the suggestions from @PamelaM.

@theboocock Are you happy to dual licence your contributions under both the _Biopython License Agreement_ AND the _BSD 3-Clause License_? See #898.

Also, are you happy to be thanked by name in the ``NEWS.rst`` and ``CONTRIB.rst`` files?

**Comment by biologyguy** [comment]
Just wanted to bump this. Is it likely to be included in the next release?

**Comment by peterjc** [comment]
Pending a reply from @theboocock

**Comment by peterjc** [comment]
Unless we hear from @theboocock, I will not merge this - we agreement on the licensing.

It will also need updating for the merge conflict and the minor issues noted in review.

**Pull request closed without merging**